README.md

```diff
@@ -70,24 +70,62 @@ for these new features.
 #### SSO Clients and LARA (authoring) integration
 
 These instructions assume that you are setting up LARA and Portal using
-docker-compose files. It *may* also assume that you are using something like Dori or Dinghy
-as an http-proxy dns container, so that the portal is available at:`//app.portal.docker`
-and lara is available at `//app.lara.docker`
+docker-compose files. It also assumes that you are using something like Dori or Dinghy
+as an http-proxy dns container, so that the portal is available at `app.portal.docker`
+and lara is available at `app.lara.docker`
 
-If you want to provide authentication services to LARA, you need to:
-1. In the Portal, edit `.env` and append `docker/dev/docker-compose-lara-proxy.yml` to the `COMPOSE_FILE` var.
-1. In the Portal, as an administrator, setup a new "Auth Client". Use the following settings:
+##### Starting from scratch on a Mac
+
+1. in the Portal: `cp .env-osx-sample .env`
+2. start up the Portal: `docker-compose up`
+3. in LARA: `cp .env-osx-sample .env`
+4. start up LARA: `docker-compose up`
+5. in the Portal, as an administrator, setup a new "Auth Client". Use the following settings:
+```
+Name: 'localhost'
+App Id: 'localhost'
+App Secret: 'unsecure local secret'
+Client Type: confidential
+Site Url: 'https://app.lara.docker' *(use https if you are running it that way...)
+Allowed Domains: (leave blank)
+Allowed URL Redirects: 'https://app.lara.docker/users/auth/cc_portal_localhost/callback'
+```
+6. If you want admin access to Lara when signing in with a portal user, you will need to first login to LARA
+with this portal user. And then either:
+    - use the rails console in LARA to set the `is_admin` flag of the newly created user.
+    - use an existing admin in LARA to make the new user an admin.
+
+##### Updating an existing setup
+
+1. In the **Portal**, edit `.env`:
+    1. Append `docker/dev/docker-compose-lara-proxy.yml` to the `COMPOSE_FILE` var.
+    2. If your portal is not running at app.portal.docker set `PORTAL_HOST`
+    3. If your portal is running on https set `PORTAL_PROTOCOL=https`
+    4. If your lara host name is not app.lara.docker, then set `LARA_HOST`
+2. Stop your portal services if they are running, and update them with `docker-compose up`
+3. In the Portal, as an administrator, setup a new "Auth Client". Use the following settings:
 ```
-Name: `localhost`
-App Id: `localhost`
+Name: 'localhost'
+App Id: 'localhost'
 App Secret: 'unsecure local secret'
-Client Type: 'confidential'
-Site Url: `https://app.lara.docker` *(use https if you are running it that way...)
+Client Type: confidential
+Site Url: 'https://app.lara.docker' *(use https if you are running it that way...)
```

Comment:
Or just run : 
`docker-compose exec app bundle exec rake sso:add_dev_client`

Reply:
That rake task doesn't set that correct values for `site_url` and `redirect_uris`.

Reply:
Oh I see you fixed that here: https://github.com/concord-consortium/rigse/pull/1064/files 

Nice! :)

I'm trying to minimize how much time I spend on this right now.  I'll try to put in a little more on it to clean it up with a reference to the rake task.